src/pages/patterns/login-pattern/index.mdx

```diff
@@ -59,104 +55,290 @@ The login page is presented to users in the following scenarios:
+| _Empty password field_                              | Password is required                 |
+| _Invalid character in an IBMid or email address_    | Enter a valid IBMid or email address |
+| _Incorrect formatting of an IBMid or email address_ | Enter a valid IBMid or email address |
+
+#### Server-side errors
+
+If there are server-side errors when the user submits the login form, the page should be reloaded, clear the password field, and return to the username input field. Use an inline notification to display the errors and provide clear direction on how users should resolve the issue. If there are multiple server errors, the inline notifications should stack.
 
 <Row>
 <Column colLg={8}>
 
-![IBMid input field](/images/Login_password_field.png)
+![Example of a server-side notificaiton on a login flow.](./images/5_Login_inline_error_736.png)
 
-<Caption>Step 2 of the login flow for non-federated users</Caption>
+<Caption>Example of a server-side notificaiton on a login flow</Caption>
 
 </Column>
 </Row>
 
-## Design types
+Incorrect username or password are the most common server errors. The application should wait until both the username and password have been submitted before checking they are valid. The same generic error message is suggested for incorrect usernames or passwords. By not telling users which field is incorrect, the application protects valid usernames from being confirmed. This is the most secure approach.
 
-The Login page should focus on the main task of accessing one’s account. The point of this page is to get the user into their account quickly without complicating the workflow. It’s not recommended to include anything that will create too much of a distraction from the form.
+The following error messages are suggested:
 
-Simple illustrations can add personality and help differentiate the form from other products, but they should not pull the user’s focus away from logging in. Consider the simplest approach to logging in to an account on mobile and maintain that simplicity on tablet and desktop.
+| Use case         | Message                                 |
+| ---------------- | --------------------------------------- |
+| _Wrong username_ | Incorrect IBMid or password, try again. |
+| _Wrong password_ | Incorrect IBMid or password, try again. |
 
-There are two design options for login pages—the single-column layout and the split-screen layout.
+## Design and layout
 
-### Single-column layout
+### Fluid vs. fixed components
+
+Although the fluid inputs have not been added as a variant to Carbon core components yet, they have complete [design specs](https://github.com/carbon-design-system/carbon/issues/5416) and are currently under development. Since many product teams have expressed interest in using the fluid inputs for Login and Sign up flows, the Carbon team wanted to consolidate exploration and present a clear path forward. What we presented above is the ideal future state of the login pattern.
```

Comment:
```suggestion
Although the fluid inputs have not been added as a variant to Carbon core components yet, they have complete [design specs](https://github.com/carbon-design-system/carbon/issues/5416) and are currently under development. Since many product teams have expressed interest in using the fluid inputs for Login and Sign up flows, the Carbon team wanted to consolidate exploration and present a clear path forward. What we have presented above is the ideal future state of the login pattern.
```